Cython/Plex/Lexicons.py

```diff
@@ -187,14 +185,13 @@ def parse_token_definition(self, token_spec):
         if not isinstance(token_spec, tuple):
             raise Errors.InvalidToken("Token definition is not a tuple")
         if len(token_spec) != 2:
-            raise Errors.InvalidToken("Wrong number of items in token definition")
+            raise Errors.InvalidToken("Wrong number of items in "
+                                      "token definition")
+
```

Comment:
Why?

Reply:
this was a line-length complaint.  will revert after updating the flake8 config to 120.